go/ql/test/library-tests/semmle/go/frameworks/StdlibTaintFlow/Fmt.go

```diff
@@ -1,4 +1,5 @@
-// Code generated by https://github.com/gagliardetto/codebox. DO NOT EDIT.
+// Code generated by https://github.com/gagliardetto/codebox, with manual additions for
+// go 1.19's new Append[f, ln] functions.
```

Comment:
Can we modify the generator instead?

Reply:
The generator is a third-party tool that we don't routinely make use of, so as of now I don't want to wade into figuring it out, updating it for Go 1.18+, etc